src/js/Keyboard.js

```diff
@@ -209,13 +209,24 @@ class Keyboard extends Component {
     }
 
     var autoCompleteWord = "";
+    var autoCompleteWordObj = {};
+    var autoCompleteList = [];
     if (this.props.autoCompleteList && this.props.autoCompleteList.length > 0) {
       const currentWord = this.state.input.split(" ").pop();
```

Comment:
```suggestion
      const currentWord = this.state.input;
```
Not certain if we should be able to chain autocomplete suggestions with this setup, seems to me we should just autocomplete based on the whole string.

Reply:
Regardless, the current setup doesn't work for suggestions which are multiple words, since the suggestions reset whenever the spacebar is used